views/agency-requests/edit_participant.php

```diff
@@ -0,0 +1,352 @@
+<?php
+/**
+ * PEP Capping 2017 Algozzine's Class
+ *
+ * Allows user to edit participant information
+ *
+ * After a user selects a participant to view, they can 
+ * edit either the participant's information.
+ *
+ * @author Vallie Joseph
+ * @copyright 2017 Marist College
+ * @version 0.7.1
+ * @since 0.6.2
+ */
+authorizedPage();
```

Comment:
Don't need this `authorizedPage()` function. This is handled in `page_permissions.ini`.